app/cu_editor.py

```diff
@@ -69,8 +69,10 @@ def mainWindowCommands(controller, textBuffer):
     KEY_ESCAPE: textBuffer.normalize,
     KEY_F1: controller.info,
     KEY_BTAB: textBuffer.unindent,
-    KEY_PAGE_UP: textBuffer.cursorPageUp,
-    KEY_PAGE_DOWN: textBuffer.cursorPageDown,
+    KEY_PAGE_UP: textBuffer.cursorNeutralPageUp,
+    KEY_PAGE_DOWN: textBuffer.cursorNeutralPageDown,
```

Comment:
Maybe we can do this for now:
```
# TODO(aaxu): Enable neutral vs. Select page up/down once we can
# differentiate those keys on Mac and Ubuntu terminals.
#KEY_PAGE_UP: textBuffer.cursorNeutralPageUp,
#KEY_PAGE_DOWN: textBuffer.cursorNeutralPageDown,
KEY_PAGE_UP: textBuffer.cursorSelectPageUp,
KEY_PAGE_DOWN: textBuffer.cursorSelectPageDown,
```

Reply:
I think we should use cursorNeutralPage for the regular keys and just keep  the cursorSelectPageUp/Down out for now. One of the reasons for this is because it was unexpected since I wasn't holding shift. The other reason is because my other PR ( #62 ) would break page down when selection mode is not none since it would keep trying to show the top of the selection.

Reply:
I see your point. In that case I'm leaning toward keeping this PR with the Neutral and Select versions intact.